content/source/docs/enterprise/api/teams.html.md

```diff
@@ -16,11 +16,19 @@ The Teams API is used to create and destroy teams. The [Team Membership API](./t
 | :----- | :------------- |
 | POST | /organizations/:organization_name/teams |
 
-### Parameters
+### URL Segments
 
-- `name` (`string: <required>`) - Specifies the name of the workspace, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the organization.
 - `:organization_name` (`string: <required>`) - Specifies the organization name under which to create the team. The organization must already exist in the system, and the user must have permissions to create new teams. This parameter is specified in the URL path.
 
+### JSON API Attributes
+
+This POST endpoint requires a JSON payload, which uses the following keys:
+
+- `data`
+    - `type` (`string: <required>`) - Must be `"teams"`.
```

Comment:
Maybe a tabular format is better for these attribute definitions. Things like defaults, required, etc, could easily be a column. The only issue there is rendering the table that people might scroll down and forget which column does which. It would at least make it possible to keep it in a standard layout.

Reply:
- The opposite of `<required>` is any kind of default value, which indicates you can omit it. 
- Yeah, no need for that to be in a code span. 👍 
- Well, I was trying to show the JSON data type. `true` vs `"teams"`. IDK, not worth it?